server/src/middlewares/timeline.test.js

```diff
@@ -0,0 +1,66 @@
+import { verifyTimelineId, verifyTimelineObject } from './timeline';
+
+describe('middleware: timeline', () => {
+  describe('verifyTimelineId: req handler create', () => {
+    let mw;
+    beforeAll(() => {
+      mw = verifyTimelineId;
+    });
+    it('should return a function', () => {
+      expect(mw.length).toBe(3);
+    });
+    it('should take 3 args', () => {
+      expect(mw).toBeInstanceOf(Function);
```

Comment:
checking for number of arguments or type of `mw` ?

Reply:
there was, as you pointed of swapped test description. fixed in latest commit

Reply:
type od middleware. swapped test description